### PR TITLE
Feature/ontology mock

### PR DESCRIPTION
**Pull request by jbout**

### CODE REVIEW

File: common/ext/class.GenerisInstaller.php [hunk 1]
```diff
@@ -54,7 +54,9 @@ public function install()
             \core_kernel_persistence_smoothsql_SmoothModel::OPTION_READABLE_MODELS => array('1'),
             \core_kernel_persistence_smoothsql_SmoothModel::OPTION_WRITEABLE_MODELS => array('1'),
             \core_kernel_persistence_smoothsql_SmoothModel::OPTION_NEW_TRIPLE_MODEL => '1',
-            \core_kernel_persistence_smoothsql_SmoothModel::OPTION_SEARCH_SERVICE => ComplexSearchService::SERVICE_ID));
+            \core_kernel_persistence_smoothsql_SmoothModel::OPTION_SEARCH_SERVICE => ComplexSearchService::SERVICE_ID,
+            \core_kernel_persistence_smoothsql_SmoothModel::OPTION_CACHE_SERVICE => common_cache_Cache::SERVICE_ID
```
Comment (hutnikau) [inline common/ext/class.GenerisInstaller.php]: Why you use service id instead of persistence id?

File: common/cache/class.NoCache.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ * 
+ * Copyright (c) 2002-2008 (original work) Public Research Centre Henri Tudor & University of Luxembourg (under the project TAO & TAO2);
```
Comment (hutnikau) [inline common/cache/class.NoCache.php]: fix license header please

File: common/oatbox/session/SessionService.php [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ * 
+ * Copyright (c) 2013 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
```
Comment (hutnikau) [inline common/oatbox/session/SessionService.php]: fix copyright year please

File: core/data/ModelManager.php [hunk 4]
```diff
@@ -32,32 +31,18 @@ class ModelManager
...
-                self::$model = self::array2model($array);
-                if (self::$model instanceof ServiceLocatorAwareInterface) {
-                    self::$model->setServiceLocator(ServiceManager::getServiceManager());
-                }
-            } else {
-                throw new \common_exception_InconsistentData('No data model found');
-            }
-        }
-        return self::$model;
+        return ServiceManager::getServiceManager()->get(Ontology::SERVICE_ID);
```
Comment (hutnikau) [inline core/data/ModelManager.php]: Did you forget to push that service file?

File: core/kernel/persistence/smoothsql/install/SmoothRdsModel.php [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+ * @author Christophe GARCIA <christopheg@taotesting.com>
+ * @license GPLv2
+ * @package generis
+ *
+ */
+namespace   oat\generis\model\kernel\persistence\smoothsql\install;
+
+use Doctrine\DBAL\Schema\Schema;
+/**
+ * Description of GateWay
```
Comment (hutnikau) [inline core/kernel/persistence/smoothsql/install/SmoothRdsModel.php]: Please fix the phpdoc and license text

File: common/ext/class.GenerisInstaller.php (hunk 1 above)
Comment (jbout) [reply common/ext/class.GenerisInstaller.php]: Using the cache abstraction instead of the persistence is closer to the original implementation

**Pull request merged**